config/services.php

```diff
@@ -231,65 +231,84 @@
     $services->alias(CollectorResolverInterface::class, CollectorResolver::class);
     $services
         ->set(AttributeCollector::class)
-        ->tag('collector', ['type' => CollectorType::TYPE_ATTRIBUTE->value]);
+        ->tag('collector', ['type' => CollectorType::TYPE_ATTRIBUTE->value])
```

Comment:
Why are 2 type tags needed here?

Reply:
An early idea was to load our Collectors via their FQCN instead of using the `type` tag. Maybe later we could implement it.